go.mod

```diff
@@ -90,22 +90,22 @@ require (
 	github.com/libp2p/go-ws-transport v0.5.0
 	github.com/miekg/dns v1.1.43
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/multiformats/go-multiaddr v0.4.0
+	github.com/multiformats/go-multiaddr v0.4.1
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/multiformats/go-multibase v0.0.3
 	github.com/multiformats/go-multicodec v0.3.0
 	github.com/multiformats/go-multihash v0.1.0
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/prometheus/statsd_exporter v0.21.0 // indirect
+	github.com/prometheus/statsd_exporter v0.22.4 // indirect
```

Comment:
I don't know if we need this to be a manual/indirect import anymore (the latest is v0.22.4). @marten-seemann why was this done originally?

Reply:
Not bumping this explicitly, and this indirect import is covered by importing `contrib.go.opencensus.io/exporter/prometheus v0.4.0`

Reply:
I’m not sure, I assume it got pulled in by either go-libp2p-quic-transport or go-tcp-transport.